<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>619</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jeopardy Creator</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="maximumSize">
       <size>
        <width>481</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="general_tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>General</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="7">
         <widget class="QLineEdit" name="game_name">
          <property name="toolTip">
           <string>Name of Your Jeopardy Game</string>
          </property>
          <property name="maxLength">
           <number>26</number>
          </property>
          <property name="placeholderText">
           <string>Game Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="row_label">
          <property name="text">
           <string>Rows:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QSpinBox" name="row_settings">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="column_label">
          <property name="text">
           <string>Columns:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="column_settings">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Default Theme: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QComboBox" name="theme_select">
          <property name="currentText">
           <string>Breath</string>
          </property>
          <item>
           <property name="text">
            <string>Breath</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dark</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Light</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Terminal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blue</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="7">
         <widget class="QPushButton" name="general_apply">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="7">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Topics</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="6">
         <widget class="QLabel" name="topic_label">
          <property name="text">
           <string>Enter only the topics for how many columns you have.</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QSpinBox" name="topic_number">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="7" column="2" colspan="5">
         <widget class="QLineEdit" name="topic_name">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Name of Topic 1</string>
          </property>
          <property name="placeholderText">
           <string>EnterTopic Name</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="7">
         <widget class="QPushButton" name="topics_apply">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Questions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0" colspan="4">
         <widget class="QLabel" name="current_ans">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QTextEdit" name="question_input">
          <property name="toolTip">
           <string>Question you want</string>
          </property>
          <property name="placeholderText">
           <string>Write Question Here</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="choose_row">
          <property name="toolTip">
           <string>Row of question you are writing</string>
          </property>
          <item>
           <property name="text">
            <string>Row 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Row 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Row 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Row 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Row 5</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="get_current">
          <property name="text">
           <string>Get Current Entry</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QLabel" name="current_que">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="choose_column">
          <property name="toolTip">
           <string>Column of question you are writing</string>
          </property>
          <item>
           <property name="text">
            <string>Column 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column 10</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="QPushButton" name="generate_button">
          <property name="toolTip">
           <string>Save the question</string>
          </property>
          <property name="text">
           <string>Generate Question</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="clear_current">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QTextEdit" name="answer_input">
          <property name="toolTip">
           <string>Answer to the question</string>
          </property>
          <property name="placeholderText">
           <string>Write Answer Here</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Slot Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Slot States:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="5" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Slot 1:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Slot 1</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Slot 2:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Pro Required</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Slot 3:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Pro Required</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="reset_button">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="5">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Detailed Slot Info:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>&lt;Name&gt;</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Rows:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>&lt;#&gt;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Columns</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>&lt;#&gt;</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Default Theme:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>&lt;#&gt;</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="5">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
